lib/apis/v3/felixconfig.go

```diff
@@ -87,6 +87,10 @@ type FelixConfigurationSpec struct {
 	// attempts to acquire the iptables lock if it is not available. Lower values make Felix more
 	// responsive when the lock is contended, but use more CPU. [Default: 50ms]
 	IptablesLockProbeInterval *metav1.Duration `json:"iptablesLockProbeInterval,omitempty" configv1timescale:"milliseconds" confignamev1:"IptablesLockProbeIntervalMillis"`
+	// FeatureDetectOverride is used to override the feature detection. Values are specified in
+	// a comma separated list. A value of "true" or "false" will force the feature, empty or omitted values
+	// are auto-detected. Features; SNATFullyRandom, MASQFullyRandom, RestoreSupportsLock
+	FeatureDetectOverride string `json:"featureDetectOverride",omitempty"`
```

Comment:
`"featureDetectOverride",omitempty"` should be be `"featureDetectOverride,omitempty"` (the omitempty is an instruction to the json parser so it needs to be inside the `json:` tag.

Reply:
Ooops, typo. Thanks for spotting it.